templates/publisher/_header.html

```diff
@@ -1,6 +1,16 @@
 {% include "publisher/_beta-notification.html" %}
 
 <section class="p-strip is-shallow u-no-padding--bottom">
+  {% if from == "fsf" %}
```

Comment:
Could it be `?from=first-snap`? It would be more readable. 'fsf' is dangerously close to FFS ;)